src/routes/docs/quick-starts/go/+page.markdoc

```diff
@@ -0,0 +1,249 @@
+
+```go
+package main
+
+import (
+	"github.com/appwrite/sdk-for-go/appwrite"
+	"github.com/appwrite/sdk-for-go/client"
+	"github.com/appwrite/sdk-for-go/databases"
+	"github.com/appwrite/sdk-for-go/models"
+)
+
+var (
+	appwriteClient    client.Client
+	todoDatabase      *models.Database
+	todoCollection    *models.Collection
+	appwriteDatabases *databases.Databases
+)
+
+func main() {
+	appwriteClient = appwrite.NewClient(
+		appwrite.WithProject("<PROJECT_KEY>"),
+		appwrite.WithKey("<API_KEY>"),
+	)
+}
+```
+
+{% /section %}
+{% section #step-5 step=5 title="Initialize database" %}
+
+Once the Appwrite Client is initialized, create a function to configure a todo collection. Import the Appwrite dependencies for id.
+
+```go
+func prepareDatabase() {
+	appwriteDatabases = appwrite.NewDatabases(appwriteClient)
+
+	todoDatabase, _ = appwriteDatabases.Create(id.Unique(), "TodosDB")
+
+	todoCollection, _ = appwriteDatabases.CreateCollection(
+		todoDatabase.Id,
+		id.Unique(),
+		"Todos")
+
+	appwriteDatabases.CreateStringAttribute(
+		todoDatabase.Id,
+		todoCollection.Id,
+		"title",
+		255,
+		true,
+	)
+
+	appwriteDatabases.CreateStringAttribute(
+		todoDatabase.Id,
+		todoCollection.Id,
+		"description",
+		255,
+		false,
+	)
+
+	appwriteDatabases.CreateBooleanAttribute(
+		todoDatabase.Id,
+		todoCollection.Id,
+		"isComplete",
+		true,
+	)
+}
+```
+
+{% /section %}
+{% section #step-6 step=6 title="Add documents" %}
+Create a function to add some mock data to your new collection.
+
+```go
+func seedDatabase() {
+	testTodo1 := map[string]interface{}{
+		"title":       "Buy apples",
+		"description": "At least 2KGs",
+		"isComplete":  true,
+	}
+
+	testTodo2 := map[string]interface{}{
+		"title":      "Wash the apples",
+		"isComplete": true,
+	}
+
+	testTodo3 := map[string]interface{}{
+		"title":       "Cut the apples",
+		"description": "Don't forget to pack them in a box",
+		"isComplete":  false,
+	}
+
+	appwriteDatabases.CreateDocument(
+		todoDatabase.Id,
+		todoCollection.Id,
+		id.Unique(),
+		testTodo1,
+	)
+
+	appwriteDatabases.CreateDocument(
+		todoDatabase.Id,
+		todoCollection.Id,
+		id.Unique(),
+		testTodo2,
+	)
+
+	appwriteDatabases.CreateDocument(
+		todoDatabase.Id,
+		todoCollection.Id,
+		id.Unique(),
+		testTodo3,
+	)
+}
+```
+
+{% /section %}
+{% section #step-7 step=7 title="Retrieve documents" %}
+
+Create a function to retrieve the mock todo data.
+
+```go
+type Todo struct {
+	Title       string `json:"title"`
+	Description string `json:"description"`
+	IsComplete  bool   `json:"isComplete"`
+}
+
+func getTodos() {
+	todos, _ := appwriteDatabases.ListDocuments(
+		todoDatabase.Id,
+		todoCollection.Id,
+		appwriteDatabases.WithListDocumentsQueries([]string{}),
+	)
+
+	for _, doc := range todos.Documents {
+		var todo Todo
+		fmt.Println(doc)
+		fmt.Printf("Title: %s\nDescription: %s\nIs Todo Complete: %t\n\n", todo.Title, todo.Description, todo.IsComplete)
+	}
```

Comment:
What did you get here? I got this:

<img width="955" alt="image" src="https://github.com/user-attachments/assets/a800e35f-5066-4995-b66a-8520c97f0ef5">

As you can see, Title, Description, and Is Todo Complete are missing or incorrect.

Reply:
If you need some guidance, refer to https://github.com/ArmanNik/MailMemo/blob/9c23dc393107e6d57c5fc375e58cfa43eb25cf9f/functions/syncCalendar/main.go#L107-L126